docs/distri/telemedida.md

```diff
@@ -111,34 +112,46 @@ iniciarà una validació per tots els comptadors de telemesura.
+1. **Neteja de duplicats**: El primer pas de la validació busca tots els perfils que
 tinguin duplicats i els elimina deixant només un dels registres.
 
-2.  **Diferents mesures**: Es comprova que no existeixin registres per la mateixa hora
+2. **Diferents mesures**: Es comprova que no existeixin registres per la mateixa hora
 i que tinguin valors de mesura diferents.
 
-3.  **Mesures impossibles**: Comprova que un perfil no tingui un valor de mesura massa
-elevat i que es consideri fora dels límits. Aquests límit es pot configurar a la variable
-**tm_profile_impossible**.
+3. **Mesures impossibles**: Comprova que un perfil no tingui un valor de mesura massa
+   elevat i que es consideri fora dels límits.
+
+    !!! Info "Variable"
+        Existeix una variable de configuració anomenada **tm_profile_impossible** que ens
+        permetrà configurar quest límit.
 
-4.  **Mesures negatives**: Assegura que no hi hagi cap perfil que contingui una valor de
+4. **Mesures negatives**: Assegura que no hi hagi cap perfil que contingui una valor de
 mesura negatiu.
 
-5.  **Bits de control**: Comprova que els registres no tinguin un valor al camp de bits
+5. **Bits de control**: Comprova que els registres no tinguin un valor al camp de bits
 de control que indiqui error.
 
-6.  **Reactiva alta**: Comprova que no hi hagi cap perfil que tingui una reactiva al
-quadrant 1 amb un valor superior a l'activa entrant. Aquesta validació es pot activar
-i desactivar amb la variable de configuració **tm_reactive_check**.
+6. **Reactiva alta**: Comprova que no hi hagi cap perfil que tingui una reactiva al
+quadrant 1 amb un valor superior a l'activa entrant.
+
+    !!! Info "Variable"
+        Existeix una variable de configuració anomenada **tm_reactive_check** que ens
+        permetrà activar i desactivar aquesta validació.
 
-7.  **Superior a la potència**: Comprova que cap perfil tingui un valor d'activa
+7. **Superior a la potència**: Comprova que cap perfil tingui un valor d'activa
 entrant superior al màxim permès segons la potència contractada al període en que es
 troba el perfil. Aquesta validació té un paràmetre de tolerància que serveix per
 configurar fins a quin percentatge de la potència màxima es considera vàlid el consum
-del perfil. Per exemple si tenim una tolerància del 120%, es permetran com a vàlids
-mesures d'activa entrant un 20% superiors a la potència total. Per configurar aquesta
-tolerància s'utilitza la variable de configuració **tm_profile_power_tolerance**. Al mateix
-temps, si el valor d'aquesta variable es configura a 0, la validació es desactiva.
+del perfil.
+
+    !!! Info "Variable"
+        Existeix una variable de configuració anomenada **tm_profile_power_tolerance** que
+        ens permetrà configurar aquesta tolerància . Al mateix temps, si el valor d'aquesta
+        variable es configura a 0, la validació es desactiva.
+
+    !!! Note "Exemple"
```

Comment:
The tranlation isn't right, replace 
`<p class=\"admonition-title\">Exemple</p>`
With 
`<p class=\"admonition-title\">Ejemplo</p>`


![image](https://user-images.githubusercontent.com/19925414/39045281-c036159c-4492-11e8-8e39-a6494300cc35.png)